docs/reference/samples/observability.output-loki-default.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: "observability.openshift.io/v1"
+kind: ClusterLogForwarder
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  outputs:
+    - name: default-lokistack
+      type: lokiStack
+      lokiStack:
+        target:
+          name: lokistack-dev
+          namespace: openshift-logging
+        authentication:
+          token:
+            from: secret
```

Comment:
It is better to use serviceAccount. here?

Reply:
Happy to use yours if you have a suggestion for a good example to use here.

Reply:
@anpingli what do you think is the most popular/useful configuration here?

Reply:
I second @anpingli 's comment and would suggest changing it to the `serviceToken`:

```
      authentication:
        token:
          from: serviceAccount
```